### PR TITLE
feat: add stacks app

### PR DESCRIPTION
**Pull request by dibyadhar**
#### 📲 What

Add terraform module to conditionally deploy DNS Zone and DynamoDB

#### 🤔 Why

It's required to deploy stacks-dotnet and stacks-dotnet-cqrs app

#### 🛠 How

Using CI-CD  tools like AIR, Github Action or Azure DevOps Pipeline 

#### 👀 Evidence

Screenshots / external resources / links / etc.
Link to documentation updated with changes impacted in the PR.

#### 🕵️ How to test

Go to example folder

Run 
```
terraform init
terraform plan
terraform apply

```
image

#### ✅ Acceptance criteria Checklist

- [ ] Code peer reviewed?
- [x] Documentation has been updated to reflect the changes?
- [x] Passing all automated tests, including a successful deployment?
- [ ] Passing any exploratory testing?
- [ ] Rebased/merged with latest changes from development and re-tested?
- [ ] Meeting the Coding Standards?

### CODE REVIEW

File: aws/modules/infrastructure_modules/stacks_app/app.tf [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Create DNS Zones
+module "zones" {
+
+  source  = "terraform-aws-modules/route53/aws//modules/zones"
```
Comment (dsmallwood-amido) [inline aws/modules/infrastructure_modules/stacks_app/app.tf:4]: 2 forward slashes here - should only be single

File: aws/modules/infrastructure_modules/stacks_app/example/main.tf [hunk 2]
```diff
@@ -0,0 +1,28 @@
+
+provider "aws" {
+}
+
+resource "random_pet" "this" {
```
Comment (dsmallwood-amido) [inline aws/modules/infrastructure_modules/stacks_app/example/main.tf:5]: Is it worth renaming this resource to be more specific? Also do we care is it's a mix of alphanumeric characters? If so, we can limit this in the resource to just numbers etc

File: aws/modules/infrastructure_modules/stacks_app/app.tf [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  version = "~> 2.0"
+  count   = var.enable_zone
+  zones   = var.public_zones
+  tags    = var.tags
+}
+
+
+# Create DynamoDB
+module "dynamodb_table" {
```
Comment (dsmallwood-amido) [inline aws/modules/infrastructure_modules/stacks_app/app.tf:13]: Do we want to specify a version for this module?

File: aws/modules/infrastructure_modules/stacks_app/app.tf (hunk 1 above)
Comment (dibyadhar) [reply aws/modules/infrastructure_modules/stacks_app/app.tf:4]: image


I guess the child modules for zones are been addressed by // in the orginal git repository
https://github.com/terraform-aws-modules/terraform-aws-route53

File: aws/modules/infrastructure_modules/stacks_app/example/main.tf (hunk 2 above)
Comment (dibyadhar) [reply aws/modules/infrastructure_modules/stacks_app/example/main.tf:5]: Yes this is not the real manifest that will be consumed, it's was just for testing out, I have in mind to write more clear examples so for time being left that out, but i can update it to random strings then pet names :-)

**Comment by ElvenSpellmaker** [comment]
@dibyadhar Make sure to end files with a newline, see the files with the no entry sign, e.g.:
image

**Comment by dibyadhar** [comment]
> @dibyadhar Make sure to end files with a newline, see the files with the no entry sign, e.g.: image

Fixed now

File: aws/modules/infrastructure_modules/stacks_app/app.tf (hunk 3 above)
Comment (dibyadhar) [reply aws/modules/infrastructure_modules/stacks_app/app.tf:12]: Updated now

**Review by dsmallwood-amido — APPROVED** [review]

**Pull request merged**